ush/prepobs_oiqcbufr.sh

```diff
@@ -78,20 +78,14 @@ TIMEIT=${TIMEIT:-""}
 #$TIMEIT mpirun -genvall -n $LSB_DJOB_NUMPROC -machinefile $LSB_DJOB_HOSTFILE $OIQCX > outout 2> errfile
 
 SITE=${SITE:-""}
-sys_tp=${sys_tp:-$(getsystem.pl -tp)}
+sys_tp=${sys_tp:-$(getsystem)}
 getsystp_err=$?
 if [ $getsystp_err -ne 0 ]; then
-   msg="***WARNING: error using getsystem.pl to determine system type and phase"
+   msg="***WARNING: error using getsystem to determine system type and phase"
    [ -n "$jlogfile" ] && $DATA/postmsg "$jlogfile" "$msg"
 fi
 echo sys_tp is set to: $sys_tp
-if [ "$sys_tp" = "Cray-XC40" -o "$SITE" = "SURGE" -o "$SITE" = "LUNA" ]; then
-   launcher_OIQCX=${launcher_OIQCX:-"aprun -n 16 -N 16 -j 1"}  # consistent with tide/gyre
-#  launcher_OIQCX=${launcher_OIQCX:-"aprun -n 24 -N 24 -j 1"}  # slightly faster
-elif [ "$sys_tp" = Dell-p3 -o "$SITE" = VENUS -o "$SITE" = MARS ]; then
-   launcher_OIQCX=${launcher_OIQCX:-mpirun}
-else
-   launcher_OIQCX=${launcher_OIQCX:-"mpirun.lsf"}
+launcher_OIQCX=${launcher_OIQCX:-mpirun}
```

Comment:
I think this should be mpiexec. For WCOSS1 (Dell) the default was mpirun. But for WCOSS2 the default is mpiexec.

Reply:
Replaced mpirun with mpiexec. Recompiled and retested. The  prep step produced identical output to before.